src/main/kotlin/no/nav/familie/ef/mottak/service/OppgaveService.kt

```diff
@@ -148,8 +153,14 @@ class OppgaveService(private val integrasjonerClient: IntegrasjonerClient,
         }
     }
 
-    private fun utledBehandlingsmåte(søknad: Søknad) =
-            if (søknad.behandleINySaksbehandling && sakService.kanOppretteInfotrygdSak(søknad))
-                "familie-ef-sak-førstegangsbehandling"
-            else null
+    private fun utledBehandlesAvApplikasjon(søknad: Søknad): BehandlesAvApplikasjon {
+        val stønadType = dokumenttypeTilStønadType(søknad.dokumenttype) ?: return BehandlesAvApplikasjon.INFOTRYGD
+        return if (søknad.behandleINySaksbehandling && sakService.kanOppretteInfotrygdSak(søknad)) {
+            BehandlesAvApplikasjon.EF_SAK_INFOTRYGD
+        } else if (saksbehandlingClient.finnesBehandlingForPerson(stønadType, søknad.fnr)) {
+            BehandlesAvApplikasjon.EF_SAK
+        } else {
+            BehandlesAvApplikasjon.INFOTRYGD
+        }
```

Comment:
Denne føles vel kanskje litt feil ut. Burde vel FØRST sjekke om det finnes en behandling for personen i ef-sak?

Reply:
Ja, tror finnesBehandlingForperson bør være først i if-sekvensen.